phone_verify/services.py

```diff
@@ -27,10 +27,13 @@ class PhoneVerificationService(object):
 
     verification_message = phone_settings.get("MESSAGE", DEFAULT_MESSAGE)
 
-    def __init__(self, phone_number, backend=None):
+    def __init__(self, phone_number, backend=None, verification_message=None):
         self._check_required_settings()
         if backend is None:
             self.backend = get_sms_backend(phone_number=phone_number)
+            
+        if verification_message is not None:
+            self.verification_message = verification_message
 
```

Comment:
In order for this to be extensible, I'd suggest picking it from the settings of the project. Yet, for the actual sending part, provide option to override the earlier setting.

Reply:
This also needs to be passed to `send_message` which calls `_generate_message` to generate actual message. This would also need documentation for end users to know how to support this custom message fromatting.